scripts/wheels/setup.py

```diff
@@ -39,6 +39,31 @@ def replaceKeywords(stringArray):
     f.writelines(list(map(lambda x: replaceKeyword(x) + "\n", conf["excluded_binaries"])))
     f.close()
 
+package_classifiers = [
+        "Programming Language :: Python :: 3",
+        "Programming Language :: C++",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.6",
```

Comment:
Idem pybind11 is stating with 3.7

Reply:
Ok, I will update for all packages after merging this :+1:

Reply:
But we are still able to generate 3.6 packages correctly, what is the problem?

Reply:
If it works, it works. But there is no official support anymore